cli/wallet/file/file.go

```diff
@@ -183,6 +179,10 @@ func (wf *fileWalletFactory) Kind() string {
 	return Kind
 }
 
+func (wf *fileWalletFactory) PrettyKind(config map[string]interface{}) string {
+	return fmt.Sprintf("%s (%s)", Kind, config["algorithm"])
```

Comment:
You should use `cfg, err := wf.unmarshalConfig(rawCfg)` and then use `cfg.Algorithm` instead of hardcoding the serialized key here. See below for other uses.